core/src/apps/solana/predefined_transaction.py

```diff
@@ -1,7 +1,9 @@
 from typing import TYPE_CHECKING
 
+from src.apps.solana.types import AdditionalTxInfo
```

Comment:
This throws an error. And it seems it can be moved under `if TYPE_CHECKING:`.

```suggestion
from .types import AdditionalTxInfo
```